packages/design-system/src/components/contextMenu/components/icon.js

```diff
@@ -26,6 +26,13 @@ import { VisuallyHidden } from '../../visuallyHidden';
 const IconWrapper = styled.div`
   width: 32px;
   height: 32px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  > * {
+    flex: 1;
+  }
```

Comment:
This allows using icons that are smaller than 32x32 while keeping them centered within `IconWrapper`.